src/ngraph/runtime/cpu/quantization_util.hpp

```diff
@@ -89,7 +89,32 @@ namespace ngraph
                     quant_util.push_back(scale);
                 }
 
-                float get_scale(const ngraph::Node* node);
```

Comment:
can you add some comments or refer to link regarding this scale/offset min/max calculation?. It will be useful to understand the difference with TF/Mxnet choice of doing v/s the ngraph.